qadence/ml_tools/saveload.py

```diff
@@ -63,8 +73,12 @@ def write_checkpoint(
     try:
         # We extract the device from the pyqtorch native circuit
         device = str(model.device).split(":")[0]  # in case of using several CUDA devices
-    except Exception:
-        pass
+    except Exception as e:
+        msg = (
+            f"Unable to identify in which device the QuantumModel is stored due to {e}."
+            + "Setting device to None"
```

Comment:
I think you can do a multiline string without the need of the + here.

Reply:
True, thanks!